tests/unit/entityInfo.spec.ts

```diff
@@ -26,10 +29,17 @@ describe('Entity Info component', () => {
     expect(wrapper.vm.$el.querySelector('#nr-header').style.display).toBe('none')
   })
 
-  it('renders the Name Request header when the NR Data is present and not the initial header', () => {
-    wrapper.vm.nameReqData = true
+  it('renders the Name Request header when the EntityType(BC) is present and not the initial header', () => {
```

Comment:
Might as well update the test description in line 26.

Reply:
good idea! Done 👍